### PR TITLE
Add initial readme text

### PR DESCRIPTION
**Pull request by vadymmarkov**
@zenangst @RamonGilabert @onmyway133 
Want to have several iterations of writing the README, so this is the first one.

### CODE REVIEW

**Review by zenangst — APPROVED** [review]
This is a great README, terrific job @vadymmarkov.
The only thing that I was wondering about is the annotation style that you use when mentioning a specific component. You went with `italic` instead of `bold`, was that intentional ?

This is a minor thing, just something that I was wondering about.

File: README.md [hunk 1]
```diff
@@ -38,6 +97,13 @@ github "hyperoslo/Aftermath"
 
 Hyper Interaktiv AS, ios@hyper.no
 
+# Influences
+
+**Aftermath** is inspired by the idea of unidirectional data flow in
```
Comment (onmyway133) [inline README.md:103]: Should we mention `influences` once?

Comment (vadymmarkov) [reply README.md:103]: Kinda want to have a separate section for that, but it's also good to mention it in description as well to let people understand what is it about.

**Comment by vadymmarkov** [comment]
@zenangst Fixed formatting.

**Comment by onmyway133** [comment]
🚀

**Pull request merged**